### PR TITLE
Set release highlights for 0.42.0

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: releaseDetails.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-    "version": "v0.40.0",
+    "version": "v0.42.0",
     "title": "RELEASE HIGHLIGHTS",
-    "highlight1": "Circom v2.1.6 support",
-    "highlight2": "Improved & stable support for Vyper development",
-    "highlight3": "New workspace template: Uniswap HookBook MultiSigSwapHook",
+    "highlight1": "Ask GPT for circom ZKP compiler errors/warnings",
+    "highlight2": "Default Solidity compiler set to 0.8.24 with support of 'Cancun' EVM",
+    "highlight3": "Workspace Template 'Uniswap v4 Periphery' renamed to 'Uniswap v4 Template'",
```
Comment (yann300) [inline releaseDetails.json:6]: This should be just `Uniswap v4: workspace template available`

Comment (Aniket-Engg) [reply releaseDetails.json:6]: I think they are are available for some time now. for v0.40.0,  the highlight was already `New workspace template: Uniswap HookBook MultiSigSwapHook`

Comment (yann300) [reply releaseDetails.json:6]: yes I understand just that `Workspace Template 'Uniswap v4 Periphery' renamed to 'Uniswap v4 Template'` isn't like a new features, it's mode like a fix.. not sure it worth mentionning it. but that's fine.

**Review by yann300 — APPROVED** [review]

File: releaseDetails.json (hunk 1 above)
Comment (Aniket-Engg) [reply releaseDetails.json:6]: This is more like a highlight not the features necessarily. Also, a good way to inform about the changes

**Pull request merged**